apps/desktop/src/vault/app/vault/add-edit.component.html

```diff
@@ -448,6 +448,24 @@ <h2 class="box-header">
               />
             </div>
           </div>
+          <!-- Fido2Key -->
+          <div *ngIf="cipher.type === cipherType.Fido2Key">
+            <div class="box-content-row" appBoxRow>
+              <label for="fido2KeyUsername">{{ "username" | i18n }}</label>
+              <input
+                id="fido2KeyUsername"
+                type="text"
+                name="Fido2Key.Username"
+                [(ngModel)]="cipher.fido2Key.userName"
+                appInputVerbatim
+                [readonly]="!cipher.edit && editMode"
+              />
+            </div>
+            <div class="box-content-row text-muted" appBoxRow>
+              <span class="row-label">{{ "typePasskey" | i18n }}</span>
+              {{ "dateCreated" | i18n }} {{ cipher.creationDate | date : "MM/dd/y" }}
```

Comment:
issue: date format should depend on locale
```suggestion
              {{ "dateCreated" | i18n }} {{ cipher.creationDate | date : "medium" }}
```

Reply:
The date format is based on the Figma designs as well

Reply:
Figma designs usually use `en-US` locale, I've added a comment in Figma to ask about locale-dependent dates

Reply:
Got a response, you can use the `short` format

Reply:
Yeah just saw that. I would be making the changes to the other clients as well